todo_app/data/trello_items.py

```diff
@@ -100,23 +96,18 @@ def delete_item(Selected_item,BOARD_ID,APIKey,APIToken):
         Nothing.
     """
 
-    items = get_items(BOARD_ID,APIKey,APIToken)
     query = {   
         'key': APIKey,
         'token': APIToken
     }
 
-    for findSelection in items:
-        if findSelection['id'] == int(Selected_item):
-            shortLink=findSelection['shortLink']
-            url = "https://api.trello.com/1/cards/" + shortLink
-            response = requests.request("DELETE",url,params=query)
-            break
+    url = "https://api.trello.com/1/cards/" + Selected_item
+    response = requests.request("DELETE",url,params=query)
 
     return None
 
-
-def completed_item(Selected_item,BOARD_ID,APIKey,APIToken):
+#def completed_item(Selected_item,BOARD_ID,APIKey,APIToken):
```

Comment:
Rather than comment this out, let's delete it! We can always recover the signature with Git if we need to, but as a comment it's likely this will just hang around for a long time making the codebase harder to digest